src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseIntegrationTestService.java

```diff
@@ -1701,13 +1701,13 @@ void resetTestCaseWeights_instructorInWrongCourse_forbidden() throws Exception {
     }
 
     void lockAllRepositories_asStudent_forbidden() throws Exception {
-        final var endpoint = ProgrammingExerciseResourceEndpoints.LOCK_ALL_REPOSITORIES.replace("{exerciseId}", String.valueOf(programmingExercise.getId()));
-        request.put(ROOT + endpoint, null, HttpStatus.FORBIDDEN);
+        final var endpoint = "/programming-exercises/" + programmingExercise.getId() + "/lock-all-repositories";
+        request.post(ROOT + endpoint, null, HttpStatus.FORBIDDEN);
     }
 
```

Comment:
What was the reason for this change? I guess the new version is easier to read, right? Same question of course for all the analogous appearances of this change.

Reply:
My request: https://github.com/ls1intum/Artemis/pull/8212#discussion_r1544883974
And yes. It is easier to read and more accessible to search for the endpoint

Reply:
Alright, makes a lot of sense, thanks 👍